content/docs/1.5/scalers/mongodb.md

```diff
@@ -0,0 +1,115 @@
++++
+title = "MongoDB"
+layout = "scaler"
+maintainer = "Community"
+description = "Scale applications based on MongoDB."
+availability = "v2.1+"
+go_file = "mongodb_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `mongodb` trigger that scales based on result of MongoDB query.
+
+```yaml
+triggers:
+  - type: mongodb
+    metadata:
+      # Required: database name
+      dbName: "test"
+      # Required: collection name
+      collection: "test_collection"
+      # Required: query expr, used by filter data
+      query: '{"region":"eu-1","state":"running","plan":"planA"}'
+      # Required: according to the number of query result, to scale job
+      queryValue: "1"
+      # Optional: This value will be assign to the metric
+      metricName: "global_metric"
+```
+
+**Parameter list:**
+
+- `dbName`  - Name of the database
+- `collection` - Name of the collection
+- `query` - A MongoDB query that should return single numeric value
+- `queryValue` - A threshold that will define when scaling should occur
+- `metricName` - An optional name to assign to the metric.
```

Comment:
Information about `metricName` is missing, please take a look on example info that's provided by other scalers:
https://keda.sh/docs/2.1/scalers/influxdb/ and https://keda.sh/docs/2.1/scalers/postgresql/